docs/security-features/network/open-ports.rst

```diff
@@ -1,11 +1,68 @@
 No Open Ports
+=============
+
+A port bound to a service that is actively listening for incoming packets on 
+a network is referred to as an "open port". Exposing open ports on untrusted 
+networks can pose a security risk if the listening service has a vulnerability 
+or is misconfigured.
+
+Historically, Ubuntu adhered to a "No Open Ports" policy, which 
+states that default installations of Ubuntu must have no listening network 
+services after initial install. Ubuntu continues to adhere as closely as possible 
+to this policy, with some exceptions. 
+
+The `"DefaultNetworkServices" policy 
+<https://wiki.ubuntu.com/DefaultNetworkServices>`_ provides detailed 
+and structured guidelines on when it is acceptable to expose network services 
+on default installations of Ubuntu. Based on the DefaultNetworkServices policy, 
+any software that accepts packets from the network and is enabled as part of the 
+default Ubuntu installation must:
+
+* not expose any application-level data to the network (unless the user explicitly requests it using an understandable interface).
+
+* confine its privileges to limit the impact of any potential arbitrary code execution vulnerabilities.
+
+* be free from problems such as lack of code quality, poor upstream security response processes, and others that would make it difficult to identify and fix security issues in the software.
+
+The network services exposed on default installations are kept to a minimum, and
+are generally limited only to services that provide substantial improvement to 
+out-of-the-box user experience. Some notable exceptions to the 
+"No Open Ports" policy are given below.
+
+Exceptions to the "No Open Ports" Policy
+++++++++++++++++++++++++++++++++++++++++
+
+Ubuntu Desktop
+--------------
+
+For Ubuntu Desktop systems, exceptions include network infrastructure services
+such as a DHCP client and the Avahi mDNS/DNS-SD daemon.
```

Comment:
I think this could be slightly expanded. Maybe explaining why these are granted exceptions would be helpful. 

DHCP is needed so it can receive incoming communication from a router to automatically obtain a network address (needed otherwise a user couldn't connect to a network). mDNS/DNS-SD daemon is needed for device discovery over a network.

So something short, but still just a bit more in-depth.